pkgs/development/python-modules/Babel/default.nix

```diff
@@ -22,6 +22,15 @@ buildPythonPackage rec {
     # https://github.com/NixOS/nixpkgs/issues/74904 (like above).
     && !(stdenv.hostPlatform.isMusl && !isPy3k);
 
+  # Without this, the build oddly fails on ofborg CI (but not on nh2's NixOS)
+  # when Python M 3 is used.
+  # Even more oddly, using `export LC_ALL="C.UTF-8"` (with quotes) makes it
+  # fail as well.
+  # See https://github.com/NixOS/nixpkgs/pull/75676#issuecomment-567289105
+  preCheck = if isPy3k then null else ''
+    export LC_ALL=C.UTF-8
+  '';
```

Comment:
could also be:
```
  preCheck = lib.optionalString isPy27 ''
    export LC_ALL=C.UTF-8
  '';

Reply:
I have no opinion on that; I wanted to express that "versions < 3" get this wrong, and `isPy27` seems more specific than that. But at the same time, is the only < 3 version available in nixpkgs and no 2.8 is supposed to be released, so it may as well be equivalent.

Whichever you think is better!

Reply:
I don't have a strong opinion either